vscode-lean4/src/utils/logger.ts

```diff
@@ -0,0 +1,18 @@
+import { Console } from 'console'
+
+class Logger extends Console {
+    constructor(stdout: NodeJS.WritableStream, stderr?: NodeJS.WritableStream) {
+        super(stdout, stderr);
+    }
+
+    log(msg: string) {
+        super.log(new Date().toLocaleTimeString(), '-', msg);
```

Comment:
```suggestion
        super.log(new Date().toLocaleTimeString('en-US'), '-', msg);
```
So it doesn't depend on the runner's locale.

Reply:
let's just go with what I have now we can easily switch to something else if we need to later.